src/App.tsx

```diff
@@ -1,10 +1,47 @@
 /* eslint-disable jsx-a11y/label-has-associated-control */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { UserWarning } from './UserWarning';
-import { USER_ID } from './api/todos';
+import { getTodos, USER_ID } from './api/todos';
+import { Todo } from './types/Todo';
+import { TodoList } from './components/TodoList/TodoList';
+import { FilterTypes } from './types/FilterTypes';
+import classNames from 'classnames';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [selectedTodos, setSelectedTodos] = useState<FilterTypes>(
+    FilterTypes.All,
+  );
+  const [isErrorHidden, setIsErrorHidden] = useState(true);
+
+  const areTodosExist = todos.length !== 0;
+  const notCompletedTodos = todos.filter(todo => !todo.completed).length;
+
+  let filteredTodos = [...todos];
```

Comment:
instead create `const` variable, use `useMemo` hook and write filtering logic inside the callback